src/applications/facility-locator/config.js

```diff
@@ -154,7 +154,9 @@ export const resolveParamsWithUrl = ({
         : null,
       `page=${page}`,
       `per_page=${perPage}`,
-      facility === LocationType.VET_CENTER ? `exclude_mobile=true` : null,
+      // facility === LocationType.VET_CENTER ? `exclude_mobile=false` : null,
```

Comment:
Can we delete the commented line?

Reply:
yes, thank you, thought I cleaned that up!